sentry-rails/spec/support/test_rails_app/app.rb

```diff
@@ -74,6 +74,7 @@ def self.name
 
   app.config.hosts = nil
   app.config.secret_key_base = "test"
+  app.config.action_dispatch.trusted_proxies = ["5.5.5.5"]
```

Comment:
on the current master you can specify app configuration with `make_basic_app` as well, so I'd prefer to configure this option only when necessary. you should be able to do something like:

```ruby
before do
  make_basic_app do |config, app|
    app.config.action_dispatch.trusted_proxies = ["5.5.5.5"]
  end
end

# the test case
```